src/NuGet.Core/NuGet.Common/PathUtil/PathResolver.cs

```diff
@@ -80,6 +80,7 @@ private static Regex WildcardToRegex(string wildcard)
                 pattern = pattern
                     .Replace(@"\.\*\*", @"\.[^/.]*") // .** should not match on ../file or ./file but will match .file
                     .Replace(@"\*\*/", "(.+/)*") //For recursive wildcards /**/, include the current directory.
+                    .Replace(@"\*\*/", "([^/]+/)*?") //For recursive wildcards /**/, include the current directory.
```

Comment:
Do we need new unit test to check correctness?

Reply:
[PathResolverTests](https://github.com/NuGet/NuGet.Client/blob/97311b5cb48d38402b49fd41a6384b11db7dd453/test/NuGet.Core.Tests/NuGet.Common.Test/PathResolverTests.cs) file contains all required unit tests.